src/client/debugger/jupyter/debuggingManager.ts

```diff
@@ -232,6 +212,50 @@ export class DebuggingManager implements IExtensionSingleActivationService, IDeb
         this.runByLineInProgress.set(runningByLine).ignoreErrors();
     }
 
+    private async tryToStartDebugging(mode: KernelDebugMode, editor?: NotebookEditor, cell?: NotebookCell) {
+        if (editor) {
+            if (this.notebookInProgress.has(editor.document)) {
```

Comment:
This is checking the case where debugging is starting, but doesn't it still need to check `this.notebookToDebugger.has`?